### ISSUE TITLE
Question: does get_running_records() really gets ALLE running tasks?

### ISSUE THREAD

**Issue by Tagirijus** [almarklein/timetagger_cli#20]
Hey almar,

just a quick question, since I do not quite understand the logic behind it. Here ...

https://github.com/almarklein/timetagger_cli/blob/f3b7a29f00e9b153d0c7953fda7afad8b0ad75d8/timetagger_cli/core.py#L65-L66

... you get the _running records_ by getting records between 35 minutes ago and in 1 minute, right? What about tasks, which started earlier?

Thanks for taking time explaining! (=

**Comment by almarklein** [comment]
Good catch! I think you are correct and that it will fail to find records that started earlier that 35 min ago. Also don't remember what the 35 was about, seems arbitrary. Better make this one week or so.
